packages/components/organisms/f-footer/src/tenants/en-AU.js

```diff
@@ -149,6 +149,11 @@ export default {
         brands: {
             title: 'Brands',
             links: [
+                {
+                    url: '/takeaway/brands/mcdonalds',
```

Comment:
Was curious about whether this should be `/content/` rather than `/takeaway/` as per the other brands - found that the content link automatically 301 redirects to the takeaway one - but that also holds for the other brands.

Should the other brands therefore be updated to reference `/takeaway/` for consistency?

Reply:
I have checked all links and updated accordingly. Thank you for that @benwhite-deltas.